components/favourites.js

```diff
@@ -90,42 +82,29 @@ export class Favourites extends Component {
       this.props.favouriteBenefits
     );
 
+    const breadcrumbs = [
+      { url: "/benefits-directory", name: t("titles.all_benefits") }
```

Comment:
if you make some radio selections on /benefits-directory, click the button to go to the favourites page, and then click the "all benefits" breadcrumb, the url query parameters for the selections are now gone. I think if you use the `common/mutate_url` function, that will fix the issue.